### PR TITLE
✨ SVK-Native Ads. Network integration

### PR DESCRIPTION
**Pull request by vskvortsov**
Integration of SVK Native Ads. content discovery platform as <amp-embed>

### CODE REVIEW

File: ads/svknative.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```
Comment (torch2424) [inline ads/svknative.md]: Incorrect Copyright Year 😄

**Comment by vskvortsov** [comment]
> Just a few small changes, thanks for the contribution! 😄 👍

Thanks for your comment, updated copyright year :)

**Comment by torch2424** [comment]
@vskvortsov 

Hello!

So code looks great, just one last thing I noticed:

When testing the ad, everything looked great! 😄 However, it seems like renderStart is not implemented properly on your end?

Screen Shot 2019-04-08 at 5 11 46 PM

Screen Shot 2019-04-08 at 5 11 55 PM

I added the code snippet to `amp-ad-xorigin-iframe-handler.js` and it seems like renderStart never gets called. However, you have `renderStartImplemented` set to true in your config.

Let me know if this is expected, or if you are still finishing the implementation.

Thank you! 😄

**Comment by vskvortsov** [comment]
@torch2424 
You are right, we haven't implemented it properly.
This is fixed now, I've just tested it the same way you did.

Please check.
Note - it required changes to our embed.js file, make sure to drop browser cache, thank you!

**Comment by torch2424** [comment]
@vskvortsov 

Awesome! Thanks for going ahead and fixing that, and confirming that for us!

I also got it working now:

Screen Shot 2019-04-10 at 2 24 12 PM

Merging this now! 😄 🎉 

This should be in PROD in about two weeks, and then your users can start using it then 😄

Thanks again for the contribution! 👍

**Comment by vskvortsov** [comment]
@torch2424 thank you for support ! 👍

**Pull request merged**